packages/next-plugin-sentry/src/on-init-client.js

```diff
@@ -0,0 +1,5 @@
+import '../../../../sentry.client.config.js';
+
+export default async function initClient() {
+  // Client initialization code
```

Comment:
I think this file is necessary mostly for the side-effect of importing the `sentry.client.config.js` file.
The comment could be more useful, there is no "initialization code" here.

Same for the server file.